recipes/llvm-core/all/conanfile.py

```diff
@@ -184,11 +184,13 @@ def requirements(self):
             self.requires('libxml2/2.9.10')
 
     def configure(self):
-        if self.options.shared:
-            del self.options.fPIC
-        if self.settings.os == 'Windows' and self.options.shared:
-            message = 'Shared build not supported on Windows'
+        if self.options.shared:  # Shared builds disabled just due to the CI
+            message = 'Shared builds not currently supported'
             raise ConanInvalidConfiguration(message)
```

Comment:
I didn't remove the code to build the package as a shared library but to raise an invalid configuration exception in all cases instead. In case someone wants to tweak the package to recover the lost functionality or, for us in the future, it is as easy as to remove this two lines and uncomment the following ones.